python/lsst/ts/observatory/control/data/atcalsys.yaml

```diff
@@ -76,3 +76,251 @@ at_whitelight_r:
     - 6.0
     - 6.0
 
+ptc_1:
+  calib_type: WhiteLight
+  use_camera: true
+  atspec_filter: SDSSr_65mm
+  atspec_grating: empty_1
+  wavelength: 500.0
+  monochromator_grating: RED
+  exit_slit: 5.0
+  entrance_slit: 5.0
+  electrometer_integration_time: 0.1
+  electrometer_mode: CURRENT
+  electrometer_range: -1
+  use_fiberspectrograph: true
```

Comment:
These two flags are currently not used, but it is good to add them here as future proofing the configuration. 

A couple suggestions;

1. Can you please move these two to be close to `use_camera` above?
2. I think we want `use_fiberspectrograph: false`
3. Add the same fields to the other configurations.